test/evergreen/evg_cfg.py

```diff
@@ -46,13 +39,6 @@
   {progname} check [-t <test_type>] [-v]
   {progname} generate [-t <test_type>] [-v]
   {progname} (-h | --help)
-
-Options:
-  -h --help     Show this screen.
-  -t TEST_TYPE  The type of test to be checked/generated.
-  -v            Enable verbose logging.
-  check         Check if any missing tests that should be added into Evergreen configuration.
```

Comment:
It looks like we've lost the descriptions of `check` and `generate` in the new help text.
If we want to do this via `argparse` we can create a [subparser](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_subparsers) but I'm ok with just keeping these lines in `USAGE`.